layouts/partner/aws.html

```diff
@@ -281,6 +208,115 @@ <h3 class="mt-12 mb-8 text-center">
+                {{ partial "code" (dict "code" .Params.automation_api_code "lang" "js" "mode" "dark" "chrome" true) }}
+            </div>
+        </div>
+    </section>
+
+    <!-- AWSX -->
+    <section id="awsx" class="container mx-auto mb-12">
+        <div class="flex flex-col lg:flex-row">
+            <div class="w-full lg:w-1/2">
+                {{ partial "code" (dict "code" .Params.awsx_code "lang" "js" "mode" "dark" "chrome" true) }}
+            </div>
+
+            <div class="w-full lg:w-1/2 px-8">
+                <p class="text-3xl font-bold inline-block">AWS Crosswalk</p>
+                <p class="mt-0">
+                    Pulumi Crosswalk for AWS is a collection of libraries that use automatic
+                    well-architected best practices to make common infrastructure-as-code
+                    tasks in AWS easier and more secure. Secure and cost-conscious defaults are chosen
+                    so that simple programs automatically use best practices for the underlying
+                    infrastructure, enabling better productivity with confidence.
+                </p>
+
+                <p class="text-lg font-bold">Application and Infrastructure Together</p>
+                <p>
+                    Crosswalk enables you to blur the lines between application and infrastructure code enabling
+                    you to author an entire full stack application in one program. With support for inline Lambda
+                    functions and ease-of-use helper functions, building robust applications on AWS has never
+                    been easier.
+                </p>
+                <p class="my-4 mt-8 text-left w-full">
+                    <a class="inline-flex items-center text-gray-700 hover:text-gray-800 font-bold text-xl" href="{{ relref . "/docs/guides/crosswalk/aws" }}">
+                        <span class="inline-block w-6 text-orange-300 mr-3">
+                            <svg aria-hidden="true" focusable="false" data-prefix="fad" data-icon="sparkles" class="svg-inline--fa fa-sparkles fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><g class="fa-group"><path class="fa-secondary" fill="currentColor" d="M423.16 186.58L448 127l59.58-24.84a8 8 0 0 0 0-14.32L448 63 423.16 3.42a8 8 0 0 0-14.32 0L384 63l-59.58 24.84a8 8 0 0 0 0 14.32L384 127l24.84 59.58a8 8 0 0 0 14.32 0zm-14.32 136.84L384 383l-59.58 24.84a8 8 0 0 0 0 14.32L384 447l24.84 59.58a8 8 0 0 0 14.32 0L448 447l59.58-24.84a8 8 0 0 0 0-14.32L448 383l-24.84-59.58a8 8 0 0 0-14.32 0z" opacity="0.4"></path><path class="fa-primary" fill="currentColor" d="M384 254.64a16.06 16.06 0 0 0-8.84-14.33l-112.57-56.39-56.28-112.77c-5.44-10.87-23.19-10.87-28.62 0l-56.28 112.77L8.84 240.31a16 16 0 0 0 0 28.67l112.57 56.39 56.28 112.77a16 16 0 0 0 28.62 0l56.28-112.77L375.16 269a16.07 16.07 0 0 0 8.84-14.36z"></path></g></svg>
+                        </span>
+                        <span class="mr-2 hover:mr-3 transition-all">
```

Comment:
If you look around the code, you'll see that we almost always use even-numbered margins and padding. We should stick this wherever possible for visual consistency. For example, this is too narrow on both sides (it's outside this PR, but taken from this page):

<img width="446" alt="image" src="https://user-images.githubusercontent.com/274700/104377585-989e1500-54db-11eb-93ee-93701e52e762.png">

These should be `p-4` or `m-4` or the like. If you could go through and make sure we're being consistent here, I'd really appreciate it; I can't really keep a close watch on all of these, so I've tried to be as consistent as possible in the code throughout the site, hoping we'd use that for reference.

Reply:
👍